umodbus/const.py

```diff
@@ -137,6 +137,9 @@
     0x4100, 0x81C1, 0x8081, 0x4040
 )
 
+#: Broadcast address
+BROADCAST_ADDR = const(0x00)
```

Comment:
Can you please move this new part up to around line 105, just to not mix up any non existing order ;) I still want to keep the CRC part at the end of the file